webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -169,8 +169,8 @@ class ManageManifestModal extends Component {
                     <FormControl
                       id="cdnUrl"
                       type="text"
-                      value={this.state.redhat_repository_url || organization.redhat_repository_url || ''}
-                      onChange={this.updateRepositoryUrl}
+                      defaultValue={this.state.redhat_repository_url || organization.redhat_repository_url || ''}
+                      onBlur={this.updateRepositoryUrl}
```

Comment:
why the change to `onBlur`? It seems to be working fine, just curious though

Reply:
Because it was updating on every change to the field and that was causing a weird issue where you couldn't update the field because of the delay in the update.